common/lib/permissions.js

```diff
@@ -43,6 +43,21 @@ const contractDeliveryManagerPermissions = [
   'youth_risk_assessment:print',
 ]
 
+const readOnlyPermissions = [
```

Comment:
Could we add a test to the [user library test](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/baec5e26e3dfb411ef2abaa7880f745cf7e437e6/common/lib/user.test.js#L121) (which is currently where these are unit tested).

Reply:
Oops, I searched for "contract_delivery_manager" and concluded that there were no tests :D